new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -222,6 +223,7 @@ public static boolean deleteNamespace(String namespace) throws ApiException {
 
   // ------------------------ Docker image  -------------------------
 
+
```

Comment:
Two blank lines?  All of your other breaks like this only have one blank line before and after.

Reply:
fixed